counsel.el

```diff
@@ -3983,16 +3897,91 @@ TREEP is used to expand internal nodes."
     (counsel-imenu)))
 
 ;;** `counsel-outline'
+(defcustom counsel-outline-display-style 'path
+  "The style used when displaying matched outline headings.
+
+If headline, the title is displayed with leading stars indicating the outline level.
+
+If path, the path hierarchy is displayed.  For each entry the title is shown.
+`counsel-outline-path-separator' is used as separator between entries.
+
+If title or any other value, only the title of the heading is displayed.
+
+For org-mode buffers, use `counsel-org-headline-display-tags' and
+`counsel-org-headline-display-todo' to display tags and todo
+keywords, respectively."
+  :type '(choice
+          (const :tag "Title only" title)
+          (const :tag "Headline" headline)
+          (const :tag "Path" path))
+  :group 'ivy)
+
+(defcustom counsel-outline-path-separator "/"
+  "Character(s) to separate path entries in matched outline headings.
+
+This variable has no effect unless `counsel-outline-display-style' is
+set to path."
+  :type 'string
+  :group 'ivy)
+
+(defcustom counsel-outline-face-style nil
+  "The face used for displaying outline headings.
+
+If org, the default faces from `org-mode' are applied, i.e. org-level-1
+through org-level-8.  Note that no cycling is in effect, therefore headings
+on levels 9 and higher will not be styled.
+
+If verbatim, the face used in the buffer is applied.  For simple
+headlines in org-mode buffers, this is usually the same as org
+except that it depends on how much of the buffer has been
+completely loaded.  If your buffer exceeds a certain size,
```

Comment:
I know this wasn't changed in this PR, but since I'm working on improving these docs, I'd like to clarify - instead of "loaded", should this say "fontified" or similar instead? Because I don't understand how a buffer can be partially loaded.